src/net_processing.cpp

```diff
@@ -2553,9 +2553,7 @@ void ProcessMessage(
     }
 
     if (!pfrom.fSuccessfullyConnected) {
-        // Must have a verack message before anything else
-        LOCK(cs_main);
-        Misbehaving(pfrom.GetId(), 1, "non-verack message before version handshake");
+        LogPrint(BCLog::NET, "Unknown command \"%s\" prior to verack from peer=%d\n", SanitizeString(msg_type), pfrom.GetId());
```

Comment:
Couple of suggestions:

- we generally refer to "message types" rather than "commands" these days.
- the message type may be known, just not supported prior to a verack (eg an "addr" message)

Suggested wording: "Unsupported message \"%s\" prior to ... "

Reply:
Shed painted.